### ISSUE TITLE
[PAGE ISSUE]: 'Flutter SDK releases' - All Lists Are Duplicated

### ISSUE THREAD

**Issue by creativecreatorormaybenot** [flutter/website#6591]
### Page URL

https://docs.flutter.dev/development/tools/sdk/releases/

### Page source

https://github.com/flutter/website/tree/main/src/development/tools/sdk/releases.md

### Describe the problem

If you scroll through **any** of the lists on the **Flutter SDK releases** page, you will notice that **every** single list item is duplicated.

https://user-images.githubusercontent.com/19204050/146781500-36dbd07c-a2cc-4bac-8f3d-bc7d576ceeda.mov

Basically, the list is displayed as:

```
...list,
...list,
```

Or:

```
A,
B,
...,
F,
G,
A,
B,
...,
F,
G,
```

### Expected fix

Every list should only contain every item once and not twice.

### Additional context

It will probably be enough to just not display the list twice.

**Comment by TahaTesser** [comment]
duplicate of https://github.com/flutter/website/issues/6329
